aim/agent/aid/universes/aim_universe.py

```diff
@@ -98,55 +98,21 @@ def get_resources(self, resource_keys):
                       resource_keys)
         result = []
         id_set = set()
-
-        for key in resource_keys:
-            fault_code = None
-            dissected = self._dissect_key(key)
-            if dissected[0] == ACI_FAULT:
-                fault_code = dissected[1][-1]
-                dissected = self._dissect_key(key[:-1])
-                key = key[:-1]
-            aci_dn = apic_client.DNManager().build(
-                [tuple(x.split('|')) for x in key])
-            dn_mgr = apic_client.DNManager()
-            aci_klass, mos_rns = dn_mgr.aci_decompose_dn_guess(aci_dn,
-                                                               dissected[0])
-            rns = dn_mgr.filter_rns(mos_rns)
-            # TODO(amitbose) We should be using 'alt_resource' only if we don't
-            # find an AIM object by using 'resource'
-            conv_info = converter.resource_map[aci_klass][0]
-            klass = conv_info.get('alt_resource') or conv_info['resource']
-            res = klass(
-                **dict([(y, rns[x])
-                        for x, y in enumerate(klass.identity_attributes)]))
+        resources = self.get_resources_for_delete(resource_keys)
```

Comment:
Since we will now ignore 'alt_resource' from the converter map, I think there will be problems with sync-ing in cases where there are high/low-level objects.